### PR TITLE
Translated title missing on some nodes.

### PR DESCRIPTION
**Pull request by salvoscala**
In a translated node of Location, Blog or Things to Do type the title is not translated.

### CODE REVIEW

**Comment by salvoscala** [comment]
QA'd

File: modules/roomify/roomify_travel_theme/roomify_travel_theme.pages_default.inc [hunk 1]
```diff
@@ -2039,10 +2039,10 @@ function roomify_travel_theme_default_page_manager_handlers() {
     ),
   );
   $display->cache = array();
-  $display->title = '';
+  $display->title = '%node:title_field';
   $display->uuid = '9ef84618-1857-465c-a31a-7230c09fde85';
   $display->storage_type = 'page_manager';
-  $display->storage_id = 'node_view__location';
+  $display->storage_id = 'node_view__location-travel';
```
Comment (acrollet) [inline modules/roomify/roomify_travel_theme/roomify_travel_theme.pages_default.inc:2045]: @salvoscala is this anything to do with the travel theme? do we also need something for roomify bootstrap?

Comment (salvoscala) [reply modules/roomify/roomify_travel_theme/roomify_travel_theme.pages_default.inc:2045]: Nope, I tested on both themes and it works fine :)

**Review by acrollet — APPROVED** [review]

File: modules/roomify/roomify_travel_theme/roomify_travel_theme.pages_default.inc (hunk 1 above)
Comment (acrollet) [reply modules/roomify/roomify_travel_theme/roomify_travel_theme.pages_default.inc:2045]: sweet

**Pull request merged**